deploy/crds/crd-certificaterequests.yaml

```diff
@@ -1,7 +1,13 @@
+# {{- include "cert-manager.crd-check" . }}
+# START crd {{- if or .Values.crds.enabled .Values.installCRDs }}
```

Comment:
suggestion: is it worth having a custom function for the "are CRDs enabled" logic? It's repeated a few times here. Don't think it's required but might be worth it?

Reply:
I don't think I know a good way to do this in Helm. I think `template` cannot be used as input for an if statement for example.